apps/src/templates/teacherDashboard/AssignmentSelector.jsx

```diff
@@ -246,7 +247,7 @@ export default class AssignmentSelector extends Component {
   };
 
   render() {
-    const {assignments, dropdownStyle, disabled} = this.props;
+    const {assignments, dropdownStyle, disabled, newSection} = this.props;
```

Comment:
nit: `isNewSection` would be a better and more conventional name for `newSection` throughout the code, so you can tell its a boolean